save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -67,24 +67,54 @@ class FixPlugin(testConfig: TestConfig, testFiles: List<String> = emptyList()) :
             }
     }
 
-    override fun rawDiscoverTestFiles(resourceDirectories: Sequence<Path>): Sequence<List<Path>> = resourceDirectories
-        .map { FileSystem.SYSTEM.list(it) }
-        .flatMap { files ->
-            files.groupBy {
-                val matchResult = defaultResourceNamePattern.matchEntire(it.name)
-                matchResult?.groupValues?.get(1)  // this is a capture group for the start of file name
+    private fun createTestFile(path: Path): Path {
+        val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / FixPlugin::class.simpleName!!)
+
+        if (!fs.exists(tmpDir)) {
+            fs.createDirectory(tmpDir)
```

Comment:
please handle exception and do logErrorAndExit

Reply:
Created TempDirException for this